src/DotNetty.Common/Concurrency/SingleThreadEventExecutor.cs

```diff
@@ -41,6 +41,7 @@ public class SingleThreadEventExecutor : AbstractScheduledEventExecutor
         public SingleThreadEventExecutor(string threadName, TimeSpan breakoutInterval)
         {
             this.terminationCompletionSource = new TaskCompletionSource();
+            this.taskQueue = this.NewTaskQueue();
```

Comment:
A virtual method call in ctor. if you are 100% sure - let's add some comment then